plugins/processors/ec2tagger/ec2tagger.go

```diff
@@ -316,6 +316,7 @@ func (t *Tagger) Init() error {
 		}
 	}
 
+	t.Log.Infof("ec2tagger: Check ec2 metadata")
```

Comment:
😄 Sounds like this addresses #463

Reply:
👍  Thought I would address it in my PR but seems too late for me.

Reply:
I think there is room for improvement with doing #463, this log should be warning level and add more details.